src/components/banners/Banner.tsx

```diff
@@ -60,10 +69,15 @@ function getColorTheme(type: BannerProps['type'], weight: keyof ColorHue) {
 
 export default function Banner(props: BannerComponentProps) {
   const { banner, onClose } = props;
-  const { type, message, pinned, intense } = banner;
+  const { type, message, pinned, intense, showMore, showMoreLinkText, showLessLinkText, additionalMessage } = banner;
```

Comment:
What about adding default showMore and showLess link texts? They would most likely be "Show more" and "Show less", of course :)

Reply:
@chowington Yes most likely Show more/less so I have reflected your suggestion. But when I checked some sites, they usually used like "Show more >>" and "Show less <<" so I set these as default values (with >> and <<)